Where_Is_My_Wife/Assets/_Main/Scripts/Input/DestroyIncorrectUIEventModule.cs

```diff
@@ -0,0 +1,23 @@
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.InputSystem.UI;
+
+namespace WhereIsMyWife.Porting
+{
+    public class DestroyIncorrectUIEventModule : MonoBehaviour
```

Comment:
Rename to `InputSystemManager` and create a method that cotains the same it has now in `Awake` named `SelectInputSystemBasedOnPlatform`